TenureInformationApi/GlobalSuppressions.cs

```diff
@@ -5,10 +5,13 @@
 
 using System.Diagnostics.CodeAnalysis;
 
-[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.CreateEventConstants.V1_VERSION")]
-[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.CreateEventConstants.SOURCE_DOMAIN")]
-[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.CreateEventConstants.SOURCE_SYSTEM")]
-[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.UpdateEventConstants.V1_VERSION")]
-[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.UpdateEventConstants.SOURCE_DOMAIN")]
-[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.UpdateEventConstants.SOURCE_SYSTEM")]
+[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "<Pending>", Scope = "member", Target = "~F:TenureInformationApi.V1.Infrastructure.CreateTenureEventConstants.V1_VERSION")]
```

Comment:
You can just add this error code into the project once if you want....

Reply:
Could you show me an example of this?